TWLight/users/templates/users/my_library.html

```diff
@@ -50,12 +50,10 @@ <h3 class="inline-header">{% trans 'Your Library' %}</h3>
     ">
       <br>
       <div class="row row-flex">
-        {% for each_authorization in proxy_bundle_authorizations %}
-          {% for each_partner in each_authorization.partners.all %}
-            <div class="col-flex col-xs-12 col-sm-4 col-md-3">
-              {% include "users/collection_tile.html" %}
```

Comment:
the more I look at this, the more I hate `collection_tile`. really should be `resource_tile`. just for code readability since we have collections/streams. again, not really your doing.